src/Tickets/Commerce/Repositories/Tickets_Repository.php

```diff
@@ -96,6 +100,8 @@ public function __construct() {
 		$this->add_simple_meta_schema_entry( 'sale_price', Ticket::$sale_price_key );
 		$this->add_simple_meta_schema_entry( 'sale_price_start_date', Ticket::$sale_price_start_date_key );
 		$this->add_simple_meta_schema_entry( 'sale_price_end_date', Ticket::$sale_price_end_date_key );
+		$this->add_simple_meta_schema_entry( 'capacity', '_tribe_ticket_capacity' );
+		$this->add_simple_meta_schema_entry( 'attendee_collection', '_tribe_tickets_ar_iac' );
```

Comment:
Can we use a variable?

Reply:
Yes, thanks for that. Will do all of those tomorrow.